core/neat/mixins/_grid-column.scss

```diff
@@ -0,0 +1,39 @@
+@charset "UTF-8";
+/// Creates Neat a grid column of requested size.
+///
+/// @argument {number (unitless)} $width [1]
+///
+/// @argument {map} $grid [$neat-default-grid]
+///   The type of grid for this column. By default, the global
+///   `$neat-default-grid` will be used.
+///
+/// @example scss
+///   .element {
+///     @include grid-column(3);
+///   }
+///
+/// @example css
+///   .element {
+///     width: calc(25% - 25px);
+///     float: left;
+///     margin-left: 20px;
+///   }
+
+@mixin grid-column($width: 1, $grid: $neat-default-grid) {
```

Comment:
After looking in to it, leaving this default as `1`, greatly simplifies things in the short term.

Once there is some sort of grid-calc function, it will be easily edited.